spacy/pipeline/transition_parser.pyx

```diff
@@ -252,13 +252,14 @@ class Parser(TrainablePipe):
     def greedy_parse(self, docs, drop=0.):
         # TODO: Deprecated
```

Comment:
```suggestion
```
I guess this should be removed?

Reply:
I wasn't sure why it said *deprecated* and not *unimplemented*. But since they are implemented again, it makes sense to remove the comments.

Reply:
It probably should have said something like "unreimplemented" :p We had the functionality before, lost it in a refactor, but always meant to reimplement it again.